docs/fr/documentation/schedule/reference.md

```diff
@@ -111,7 +111,7 @@
  | Nom du fichier | Présence | Descriptif | 
  |------|------|------| 
  | [agency.txt](#agencytxt) | **Requis** | Agences de transport en commun dont le service est représenté dans ce jeu de données. | 
- | [stops.txt](#stopstxt) | **Requis** | Arrêts où les véhicules prennent ou déposent les passagers. Définit également les gares et les entrées des gares. | 
+ | [stops.txt](#stopstxt) | **Requis sous condition** | Arrêts où les véhicules prennent ou déposent les passagers. Définit également les gares et les entrées des gares. <br><br>Requis sous condition :<br> - Optionnel si des zones à la demande sont définies dans [locations.geojson](#locationsgeojson). <br>- **Obligatoire** sinon. | 
```

Comment:
I find this slightly confusing but the confusion comes from the English (_Optional if demand-responsive zones are defined in locations.geojson_), not the translation. The demand-responsive applies to a **service**, not a zone. It would be clearer if it was:

> Optionnel si des services de transport à la demande zonaux sont définis dans [...]

Which would need the English to be:

> Optional if zonal demand-responsive transit services are defined in [...]

Reply:
Got it! I'm leaning towards changing only the FR translation in this case as the english adjustment seems to me also not ideal.